SixTest/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+#Test that the testing tools run!
+ADD_TEST(NAME SelfTest_verify10 COMMAND verify10)
+set_tests_properties(SelfTest_verify10 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*verify10_DIFF I\\/O error, wrong no of lines\\!\\!\\?\\? line no *0")
+
+ADD_TEST(NAME SelfTest_checkf10 COMMAND checkf10)
+set_tests_properties(SelfTest_checkf10 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*checkf10_DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_compf10 COMMAND compf10)
+set_tests_properties(SelfTest_compf10 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*compf10_DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_checkf110 COMMAND checkf110)
+set_tests_properties(SelfTest_checkf110 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_checkf1014 COMMAND checkf1014)
+set_tests_properties(SelfTest_checkf1014 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*checkf1014_DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_read90 COMMAND read90)
+set_tests_properties(SelfTest_read90 PROPERTIES PASS_REGULAR_EXPRESSION "At line .* of file .*read90.f90 \\(unit = 190\\)\nFortran runtime error: Cannot open file 'fort.190': No such file or directory")
+
+SET(SIXTRACK_TESTS ${SIXTRACK_TESTS})
+#SET(SIXTRACK_TESTS bbe51 bbe52 bbe571ib0 bb bb_ntwin1 crabamp dipedge distance dynk_globalvars elensidealthck4d elensidealthck6d elensidealthin4d elensidealthin6d_DYNK elensidealthin6d eric exact fma frs60 frs javier lostevery lost lostnotilt notilt prob1 prob3 s316 thick4 thick6dblocks thick6ddynk tilt ${SIXTRACK_TESTS})
+
+##
+## Needs checking: thick6dsingles (missing files)
+## thick6dblocks elensidealthck6d dynk_globalvars thick6ddynk - empty fort.10
+## Disabled: elensidealthin6d_DYNK
+## "Totally untested": bnl
+##
+
+SET(STF_TESTS bbe51 bbe52 bbe571ib0 bb_ntwin1 bb crabamp dipedge distance elensidealthck4d elensidealthin4d elensidealthin6d eric exact fma frs60 frs javier_bignpart javier lostevery lostnotilt lost notilt prob1 prob3 s316 thick4 tilt)
+
+#only turn on the bnl test if we have BNLELENS enabled due to slight differences in the test
+if(BNLELENS)
+	SET(SIXTRACK_TESTS bnl ${SIXTRACK_TESTS})
+endif(BNLELENS)
+
+if(STF AND (BIGNPART OR HUGENPART))
```

Comment:
It does need bignpart:
```
TRACKING PARAMETERS-----------------------------------------------------
10000 0 64 0.84498 0.42249 0 1
1 1 2 1 2
0 0 1 1 1 50000 2
NEXT
```
64 pairs -> 128 particles.

However in principle it doesn't really need STF (say, if we in the future have an HDF5-based STF which is a different flag and a different canonical), however big/huge-npart can't be compiled without STF, and the test is STF compatible.